src/lib.rs

```diff
@@ -205,6 +219,106 @@ pub fn handle_transaction_internal(
     Ok((nullifiers, witnesses))
 }
 
+#[wasm_bindgen]
+pub fn create_transaction(
+    notes: JsValue,
+    extsk: &str,
+    to_address: &str,
+    change_address: &str,
+    amount: u64,
+    block_height: u32,
+    is_testnet: bool,
+) -> String {
+    let notes: Vec<(Note, String, String)> =
+        serde_wasm_bindgen::from_value(notes).expect("Cannote deserialize notes"); // Note, Witness, Address
+    let extsk = decode_extsk(extsk, is_testnet);
+
+    if is_testnet {
+        create_transaction_internal(
+            &notes,
+            &extsk,
+            to_address,
+	    change_address,
+            amount,
+            BlockHeight::from_u32(block_height),
+            TEST_NETWORK,
+        )
+        .expect("Failed to create tx")
+    } else {
+        create_transaction_internal(
+            &notes,
+            &extsk,
+            to_address,
+	    change_address,
+            amount,
+            BlockHeight::from_u32(block_height),
+            MAIN_NETWORK,
+        )
+        .expect("Failed to create tx")
+    }
```

Comment:
Minor code simplification:
```suggestion
    let network = if is_testnet { TEST_NETWORK } else { MAIN_NETWORK };
    create_transaction_internal(
        &notes,
        &extsk,
        to_address,
	    change_address,
        amount,
        BlockHeight::from_u32(block_height),
        network,
    )
    .expect("Failed to create tx")
```

Reply:
This does not work MAIN_NETWORK and TEST_NETWORK have different type

Reply:
Ah, wondered why the redundancy - resolved.

Reply:
Looks like they have a Network enum which allows to fix the redundancy